serial/serialutil.py

```diff
@@ -186,6 +186,7 @@ def __init__(self,
                  dsrdtr=False,
                  inter_byte_timeout=None,
                  exclusive=None,
+                 low_latency=None,
```

Comment:
the platform dependent `set_low_latency_mode` would be OK, but i'd prefer to have no changes on the base class as of now.

Reply:
I will make that change. Just give me a few minutes